### ISSUE TITLE
Error handling websocket connection

### ISSUE THREAD

**Issue by hustf** [piever/InteractDemos#4]
Hi,
testing the package and reporting.

Navigate to http://localhost:8000/
Initialization includes opening a websocket at wsuri "ws://localhost:8000/webio-socket"
and sending the message
"{"type":"message","scope":"scope-fa81698a-e392-49b4-92dd-594b11c07964","command":"_setup_scope","data":{}}"

When clicking on the 'text' widget:
```
julia> Error handling websocket connection:
WebSockets.WebSocketClosedError("ws|server respond to OPCODE_CLOSE 1001:Going Away")
```

The above is the server side (i.e. Julia) complaining that the other side wanted to close the websocket, following the protocol while doing so. The 'Error handling websocket connection' is made by Mux/websocket_integration.jl:13, which is a try-catch.

The lazy / macro / anonymous functions make it pretty hard to follow what happens here, exactly. We're on a high level of abstraction. Instead, I resort to following hunches, and found this line in WebIO/ generic_http:118:

        server = WebSockets.ServerWS(handler, wshandler, logger; server_kw_args...)

This will no longer work. The maintainers of WebSockets.jl were quite rude and did not provide any deprecation messages. Instead, it's documented here:
https://github.com/JuliaWeb/WebSockets.jl
##Errors after updating?
###To version 1.5.0
Keyword logger is no longer supported. For redirecting logs, use Logging.with_logger

**Comment by shashi** [comment]
Thanks for bringing this to our notice,

cc @SimonDanisch who wrote the generic_http.jl provider
